doc/examples/onnx_importer.md

```diff
@@ -0,0 +1,88 @@
+# ONNX importer usage examples
+
+This sample demonstrates how to use ONNX importer API.
+The API allows to create nGraph Function from ONNX importer model.
+
+All functions forming ONNX importer API are in [onnx.hpp][onnx_header] header file.
+The available functions can be divided into two categories from user's point of view.
+The first group contains helpers' functions used to check which ONNX op are supported in current version of ONNX importer.
+The second group contains function used to read ONNX models from stream or file. Their result is nGraph function which can be calculated on Inference Engine.
+
+## How to check which ONNX ops are supported
+
+In order to list all supported ONNX ops in a specific version and domain `get_supported_operators` function should be used.
+The code listing below shows how to do it.
+```
+const std::int64_t version = 12;
+const std::string domain = "ai.onnx";
+const std::set<std::string> supported_ops = ngraph::onnx_import::get_supported_operators(version, domain);
+
+for(const auto& op : supported_ops)
+{
+    std::cout << op << std::endl;
+}
+```
```

Comment:
After you document this code as it's supposed to be run, you can continue the .rst file to explain that the "output" a user can expect from running the file should look like the list of Ops below. 

Might say something like 

> The above code compiles to produce a list of all the supported operators for the ``version`` and ``domain`` you select.

Reply:
`The above code compiles to produce a list of all the supported operators for the version and domain you select. The output should look similar to what is below` version used